.env.sample

```diff
@@ -0,0 +1,2 @@
+GITHUB_TOKEN=
```

Comment:
It's a good practice to not leave sensitive information, such as tokens and API keys, uninitialized in your code directly. Consider using environment variables or a configuration file (.env) that is excluded from version control to securely manage these values.